<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="widget_button_compare">
           <property name="text">
            <string>Compare</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="widget_Hideme">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Hideme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="widget_checkBox_0">
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="widget_checkBox_1">
           <property name="text">
            <string>CheckBox</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="widget_checkBox_2">
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>864</width>
            <height>95</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="QGroupBox" name="widget_item_info">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>3</number>
              </property>
              <item row="0" column="0">
               <widget class="File_Text_Brower" name="widget_textBrowser_0">
                <property name="lineWrapMode">
                 <enum>QTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="File_Text_Brower" name="widget_textBrowser_1">
                <property name="lineWrapMode">
                 <enum>QTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="File_Text_Brower" name="widget_textBrowser_2">
                <property name="lineWrapMode">
                 <enum>QTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDockWidget" name="dockWidget">
      <property name="features">
       <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
      </property>
      <widget class="QWidget" name="dockWidgetContents">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextEdit" name="widget_textEdit_compare"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>File_Text_Brower</class>
   <extends>QTextEdit</extends>
   <header>Plugins.GUI.File_Viewer_Window</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
